### DISCUSSION TITLE
how to repository delete

### DISCUSSION THREAD

**Discussion by samplervjgithub** [community/community#54660] [New to GitHub]
their is unbale to delete repositary

**Comment by radar-blip** [comment]
If you want to delete a repository, 
 1. Open the repository you want to delete.
 2. Click on settings at the top of that page.
 3. Scroll all the way to bottom of page, and delete.

![repodelete](https://user-images.githubusercontent.com/39196767/236647517-b9fd0a4f-75af-41d7-98d7-627ca058a78e.png)

**Comment by samplervjgithub** [comment]
I try but this not work

**Reply by radar-blip** [reply]
Really? You must type your `username/repositoryname` in the little box. 
Maybe you made in error in the way you typed that info ?

**Comment by samplervjgithub** [comment]
Yes!
(simplervjgithub/Netflix -project)

**Comment by airtower-luna** [comment]
The way you typed it in your last post has a space. Repository names can't have spaces.

**Comment by SerageldinAyman** [comment]
Go to the repo >> click the settings >> scroll to the very end and you would see the "Danger zone" in Red color>> then choose the "Delete this repository" option then it will prompt you to confirm that you want to delete.
it will be deleted after wards Refresh the page
